profile/README.md

```diff
@@ -1,2 +1,33 @@
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
+
 # The Digital Public Goods Alliance
 ![DPG Cover Photo](https://github.com/DPGAlliance/.github/blob/main/profile/logo.png)
+
+## Overview
+
+Welcome to the Digital Public Goods Alliance Github. We are an not-for-profit, multi-stakeholder alliance that stewards the [definition of 'digital public goods'](https://www.un.org/techenvoy/content/digital-public-goods). Digital public goods (DPGs) are open-source products that advance the [sustainable development goals](https://sdgs.un.org/#goal_section). DPGs are open software, data, AI solutions as well as standards and content collections. Other than being open-source and platform independant, DPGs have to adhere to privacy and other applicable laws and best practices, do no harm by design and meet the [DPG standard](https://github.com/DPGAlliance/DPG-Standard). 
```

Comment:
@prajectory I would suggest using the same wording as the definition details for instance: `Digital public goods are open-source software, open data, open AI models, open standards and open content` versus `DPGs are open software, data, AI solutions as well as standards and content collections`